### ISSUE TITLE
lvcreate do not have force option

### ISSUE THREAD

**Issue by daixiang0** [chef-cookbooks/lvm#140]
Testing lvm on Centos7.2, it has become pickier:
```
[root@ip-rh-backend1 ~]# lvcreate -L 500g -i 1 -W y -n lv-data vg-data
WARNING: xfs signature detected on /dev/vg-data/lv-data at offset 0. Wipe it? [y/n]:
```
Why  `lvcreate` do not have `-f` so that i do not need to enter yes?
I think `lvcreate` could like 'pvcreate' or 'vgcreate' has force option.

**Comment by irvingpop** [comment]
@daixiang0 I'm not a maintainer of this cookbook and I'm no longer a user, so pinging me isn't going to get you what you want 😄

**Comment by daixiang0** [comment]
ping @tas50

**Comment by ChrisAWalker** [comment]
If you look at the code, it has a 'lv_params' where you can issue additional options to the command.  I've added it to the documentation, as it was hidden deep into the code:
https://github.com/chef-cookbooks/lvm/pull/160

**Comment by irvingpop** [comment]
Closed via #160
